src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -0,0 +1,26 @@
+import "./WhaleSightingViewer.scss";
+
+export function WhaleSightingViewer() {
+	return 	<div>
```

Comment:
Change this `<div>` to a fragment `<>`
```suggestion
	return 	<>
```